ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/maintenance.j2

```diff
@@ -0,0 +1,70 @@
+{# maintenance #}
+{% set first_line = {'flag': True} %}
```

Comment:
Impressive that you have caught all EOS behaviors for setting `!` in here. It makes it a bit cluttered, so I will ask @carlbuchmann and @titom73 for their view on this.

Reply:
We will keep current template behavior.